clients/src/main/java/org/apache/kafka/common/config/SslConfigs.java

```diff
@@ -50,7 +50,7 @@
     public static final String SSL_PROTOCOL_CONFIG = "ssl.protocol";
     public static final String SSL_PROTOCOL_DOC = "The SSL protocol used to generate the SSLContext. "
             + "Default setting is TLS, which is fine for most cases. "
-            + "Allowed values in recent JVMs are TLS, TLSv1.1 and TLSv1.2. SSL, SSLv2 and SSLv3 "
+            + "Allowed values in recent JVMs are TLS, TLSv1.1, TLSv1.2 and TLSv1.3. SSL, SSLv2 and SSLv3 "
             + "may be supported in older JVMs, but their usage is discouraged due to known security vulnerabilities.";
 
     public static final String DEFAULT_SSL_PROTOCOL = "TLS";
```

Comment:
Does this need to change as well?  Or does the enabled protocol list constrain it?

Reply:
It seems, doc inconsistent for now.
We do support TLVv1.3.

see e275742f